src/model_outputfiles.cpp

```diff
@@ -41,7 +42,7 @@ void Model::createReport(std::string path){
   output_report << "<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>\n\n";
   output_report << "Website: https://molovol.com\n";
   output_report << "Source code available at https://github.com/molovol/MoloVol under the MIT licence\n";
-  output_report << "Copyright © 2020-2022 Jasmin B. Maglic, Roy Lavendomme\n\n";
+  output_report << "Copyright © 2020-2024 Jasmin B. Maglic, Roy Lavendomme\n\n";

```

Comment:
Should but copyright 2020-2025.

Reply:
Now that the whole build process runs through cmake for all platforms, it would be possible to specify the year and version only once in the CMakeLists.txt and automatically update it in all relevant locations. I haven't done that yet because I didn't want to mess up your development environment.

Alternatively, I've had good success with simply using a search tool (grep) on mac. I can find references to the version and the year very fast. What happened here is that I probably changed it last year. I will double check the dates one more time.